### PR TITLE
Add first 200 most used words in Spanish

### PR DESCRIPTION
**Pull request by YagoGG**
This PR adds a new test file with the 200 most used words in Spanish, taken from the [RAE](http://www.rae.es/) (Royal Spanish Academia, or *Real Academia Española*), the official institution that regulates Spanish wordwide.

### CODE REVIEW

File: test/queries/spanish_01.txt [hunk 1]
```diff
@@ -0,0 +1,201 @@
+a
```
Comment (sudheesh001) [inline test/queries/spanish_01.txt]: This is probably a very vague query to make. Can you replace this with another word ?

File: test/queries/spanish_01.txt [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+dentro   
+desde     
+después         
+día      
+días   
+dijo   
+donde  
+dos    
+durante  
+e
```
Comment (sudheesh001) [inline test/queries/spanish_01.txt]: same here

File: test/queries/spanish_01.txt [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+nacional      
+nada       
+ni       
+no     
+nos       
+nuestro   
+nueva  
+nuevo      
+nunca  
+o
```
Comment (sudheesh001) [inline test/queries/spanish_01.txt]: same here again.

File: test/queries/spanish_01.txt [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+un     
+una    
+uno    
+unos       
+va     
+veces      
+ver       
+vez    
+vida   
+y
```
Comment (sudheesh001) [inline test/queries/spanish_01.txt]: same here

File: test/queries/spanish_01.txt [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+unos       
+va     
+veces      
+ver       
+vez    
+vida   
+y      
+ya       
+yo     
+
```
Comment (sudheesh001) [inline test/queries/spanish_01.txt]: remove the last line space.

**Comment by YagoGG** [comment]
I'll replace them with the next most used ones!

**Comment by YagoGG** [comment]
Done! I also replaced a couple more that weren't that relevant.

**Comment by DengYiping** [comment]
@YagoGG Maybe squash the commits.

**Comment by YagoGG** [comment]
49e013e should do the trick.

**Pull request merged**